.github/workflows/ci.yml

```diff
@@ -44,22 +44,10 @@ jobs:
       -
         name: Check HTTPS reachability
         run: curl -vk -o /dev/null https://localhost
-      -
-        name: Create test database
-        if: false # Remove this line if Doctrine ORM is installed
-        run: docker compose exec -T php bin/console -e test doctrine:database:create
-      -
-        name: Run migrations
-        if: false # Remove this line if Doctrine Migrations is installed
-        run: docker compose exec -T php bin/console -e test doctrine:migrations:migrate --no-interaction
       -
         name: Run PHPUnit
         if: false # Remove this line if PHPUnit is installed
```

Comment:
PHPUnit is installed by default with Laravel, so you can remove this line.

Reply:
Removed! thk